src/autogluon/bench/frameworks/multimodal/setup.sh

```diff
@@ -30,4 +30,4 @@ python3 -m pip install -e features/
 python3 -m pip install -e multimodal/[tests]
 
 python3 -m mim install -q mmcv-full
-python3 -m pip install "mmdet>=2.28, <3.0.0"
+python3 -m pip install mmdet
```

Comment:
Why removing the bound? Things might break if mmdet released newer version, i.e. 3.0

Reply:
Automm has just supported both `3.0` and `2.x`. I removed the bound because we don't pin the version in the [official guide](https://auto.gluon.ai/stable/tutorials/multimodal/object_detection/quick_start/quick_start_coco.html) either.

Reply:
OK. We should probably pin it in our docs too.... Otherwise the doc will likely become invalid in the future if someone is referring to a guide of an old version of autogluon.

I would still recommend pinging it here to avoid bench being broken, but feel free to sync with Haoyang regarding that.

Reply:
Good point. I just updated.